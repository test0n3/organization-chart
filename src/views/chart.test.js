import React from "react";
import { render } from "@testing-library/react";
import ChartView from "./chart";

test("ChartView component", () => {
  const { asFragment } = render(<ChartView />);

  expect(asFragment()).toMatchSnapshot();
});
