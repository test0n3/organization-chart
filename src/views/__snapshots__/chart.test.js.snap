// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChartView component 1`] = `
<DocumentFragment>
  <div>
    <div
      class="css-dvxtzn"
    >
      <article
        class="css-5c5px"
      >
        <p
          class="css-1kebmcj"
        />
        <p
          class="css-197a6t6"
        >
          CEO
        </p>
        <p
          class="css-ub9u27"
        >
          <button
            class="css-1b02a2r-InputCollaborator"
          >
            +
          </button>
        </p>
      </article>
      <div
        class="css-p58oka"
      >
        <div
          class="css-dvxtzn"
        >
          <article
            class="css-5c5px"
          >
            <p
              class="css-1kebmcj"
            >
              Mayra Navarro
            </p>
            <p
              class="css-197a6t6"
            >
              Product Manager
            </p>
            <p
              class="css-ub9u27"
            >
              <button
                class="css-1b02a2r-InputCollaborator"
              >
                +
              </button>
            </p>
          </article>
          <div
            class="css-p58oka"
          />
        </div>
        <div
          class="css-dvxtzn"
        >
          <article
            class="css-5c5px"
          >
            <p
              class="css-1kebmcj"
            >
              Deivy Conde
            </p>
            <p
              class="css-197a6t6"
            >
              Tech Lead
            </p>
            <p
              class="css-ub9u27"
            >
              <button
                class="css-1b02a2r-InputCollaborator"
              >
                +
              </button>
            </p>
          </article>
          <div
            class="css-p58oka"
          >
            <div
              class="css-dvxtzn"
            >
              <article
                class="css-5c5px"
              >
                <p
                  class="css-1kebmcj"
                >
                  Cristian Granda
                </p>
                <p
                  class="css-197a6t6"
                >
                  Tech Lead
                </p>
                <p
                  class="css-ub9u27"
                >
                  <button
                    class="css-1b02a2r-InputCollaborator"
                  >
                    +
                  </button>
                </p>
              </article>
              <div
                class="css-p58oka"
              />
            </div>
            <div
              class="css-dvxtzn"
            >
              <article
                class="css-5c5px"
              >
                <p
                  class="css-1kebmcj"
                >
                  Cesar Cachay
                </p>
                <p
                  class="css-197a6t6"
                >
                  Tech Lead
                </p>
                <p
                  class="css-ub9u27"
                >
                  <button
                    class="css-1b02a2r-InputCollaborator"
                  >
                    +
                  </button>
                </p>
              </article>
              <div
                class="css-p58oka"
              />
            </div>
          </div>
        </div>
        <div
          class="css-dvxtzn"
        >
          <article
            class="css-5c5px"
          >
            <p
              class="css-1kebmcj"
            >
              Ricardo Yrupailla
            </p>
            <p
              class="css-197a6t6"
            >
              Tech Lead
            </p>
            <p
              class="css-ub9u27"
            >
              <button
                class="css-1b02a2r-InputCollaborator"
              >
                +
              </button>
            </p>
          </article>
          <div
            class="css-p58oka"
          >
            <div
              class="css-dvxtzn"
            >
              <article
                class="css-5c5px"
              >
                <p
                  class="css-1kebmcj"
                >
                  Diego Cuevas
                </p>
                <p
                  class="css-197a6t6"
                >
                  Tech Lead
                </p>
                <p
                  class="css-ub9u27"
                >
                  <button
                    class="css-1b02a2r-InputCollaborator"
                  >
                    +
                  </button>
                </p>
              </article>
              <div
                class="css-p58oka"
              >
                <div
                  class="css-dvxtzn"
                >
                  <article
                    class="css-5c5px"
                  >
                    <p
                      class="css-1kebmcj"
                    >
                      Christopher Roa
                    </p>
                    <p
                      class="css-197a6t6"
                    >
                      Tech Lead
                    </p>
                    <p
                      class="css-ub9u27"
                    >
                      <button
                        class="css-1b02a2r-InputCollaborator"
                      >
                        +
                      </button>
                    </p>
                  </article>
                  <div
                    class="css-p58oka"
                  />
                </div>
                <div
                  class="css-dvxtzn"
                >
                  <article
                    class="css-5c5px"
                  >
                    <p
                      class="css-1kebmcj"
                    >
                      Frank Condezo
                    </p>
                    <p
                      class="css-197a6t6"
                    >
                      Tech Lead
                    </p>
                    <p
                      class="css-ub9u27"
                    >
                      <button
                        class="css-1b02a2r-InputCollaborator"
                      >
                        +
                      </button>
                    </p>
                  </article>
                  <div
                    class="css-p58oka"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
